tests_tensorflow/resources/buyer/build.js

```diff
@@ -0,0 +1,31 @@
+const tf = require('@tensorflow/tfjs-node');
+
+const fs = require('fs');
+
+[...Array(5).keys()].forEach(idx => {
+  const model = tf.sequential({
+    layers: [
+      tf.layers.dense({ inputShape: [200], units: 200, activation: 'relu', useBias: false, weights: [tf.randomNormal([200, 200])] }),
+      tf.layers.dense({ units: 100, activation: 'relu', useBias: false, weights: [tf.randomNormal([200, 100])] }),
```

Comment:
is 'generating random weights' part of generateBid()? it should be precomputed in build.js and passed to generated script